tokens/src/core/global/other.json

```diff
@@ -1,4 +1,8 @@
 {
+  "$type": "percentage",
```

Comment:
[clarification] only the `theme-interval` token specifically in this file is a percentage, right?

Reply:
Resolved in https://github.com/openedx/paragon/pull/3207